docs/automated-process.md

```diff
@@ -0,0 +1,30 @@
+# Automated Ansible Release Process
```

Comment:
We could make this a more action-oriented title. Maybe something like:
```suggestion
# How to release a new version of the Ansible Community Package - Automated Release Process
```
That gets wordy, though.

Reply:
Changed in 5ad0ad71749cb2907b8cdd6115246bbd5d9b8045.